PocketKit/Sources/PocketKit/Item/ItemListView.swift

```diff
@@ -137,6 +137,11 @@ private struct ItemListViewRow: View {
             )
                 .trackable(.home.item(index: UInt(index)))
                 .onAppear {
+                    let contexts = contexts
```

Comment:
`contexts` is a (simple) computed property. Despite it being simple, this assignment is to ensure we're only running the computation once.